dom-events/exercise-1.js

```diff
@@ -0,0 +1,85 @@
+(() => {
+  const table = document.querySelector('#row-container')
+
+  const ROW = 100
+  const COL = 100
+
+  function createARow(col, start) {
+    const tr = document.createElement('tr')
+
+    let td = null
```

Comment:
We usually don't use `null` at the time we declare variable.
Just do 
` let td`

Reply:
Already improved